docs/Simulation/FPGA-Accelerated-Simulators.rst

```diff
@@ -0,0 +1,64 @@
+FPGA-Accelerated Simulators
+==============================
+
+FireSim
+-----------------------
+
+`FireSim <https://fires.im/>`__ is an open-source cycle-accurate FPGA-accelerated full-system hardware simulation platform that runs on cloud FPGAs (Amazon EC2 F1).
```

Comment:
Have we decided if we're trying to stick with "cycle-exact" or "cycle-accurate"?